qoffer-domain/pom.xml

```diff
@@ -1,14 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xmlns="http://maven.apache.org/POM/4.0.0"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <parent>
-        <artifactId>qOffer_2.0</artifactId>
-        <groupId>life.qbic</groupId>
-        <version>1.0.0-SNAPSHOT</version>
-    </parent>
-    <modelVersion>4.0.0</modelVersion>
+  xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <artifactId>qoffer-domain</artifactId>
+  <dependencies>
+    <dependency>
+      <artifactId>data-model-lib</artifactId>
+      <groupId>life.qbic</groupId>
+      <scope>compile</scope>
+      <version>1.9.4</version>
```

Comment:
The version is to set in the parent pom otherwise we will run into transitive dependency resolvement issues.

Reply:
We need to discuss this further. Using the project's pom as `parent-pom` does not work for some reason and `data-model-lib` dependencies are not found. Furthermore, since the project's parent pom is a child of the `portlet-parent-pom` the `qoffer-domain` module tries to compile a vaadin project. This is wrong since this module does not have any `webapp` folder nor should it have a dependency on vaadin as it only contains core logic.

Reply:
is addressed in #5